package.json

```diff
@@ -15,7 +15,7 @@
     "start": "./bin/start",
     "test": "npm run unit",
     "travis": "npm test",
-    "unit": "./bin/units",
+    "unit": "PELIAS_CONFIG=./test/pelias-config-empty.json ./bin/units",
```

Comment:
This actually would work great in `./bin/units` itself (I think). That way anyone who runs the unit tests through any mechanism would get the right PELIAS_CONFIG